scripts/quarantine_no_optimization.yaml

```diff
@@ -1,8 +0,0 @@
-# The configurations resulting as a product of scenarios and platforms
-# will be skipped if quarantine is used. More details here:
-# https://docs.zephyrproject.org/latest/guides/test/twister.html#quarantine
-# To have an empty list use:
-- scenarios:
-    - net.lib.aws_iot
-  platforms:
-    - native_posix
```

Comment:
Please, don't remove this file, as it is now required by the plan. Clear it out instead by replacing this entry with:
```
- scenarios:
    - None
```

Reply:
Thanks. I was unsure what to do. I will fix as you suggest.

Reply:
@PerMac  Fixed. Also updated the comment in the header of the quarantine file. Please have a look.